code/game/objects/items/toys.dm

```diff
@@ -215,6 +215,9 @@
 
 /obj/item/toy/ammo/gun/examine(mob/user)
 	..()
+	if (src.amount_left == 0) {
+		return
+	}
```

Comment:
Braces. But also is this really necessary? What's wrong with "There are 0 caps left."? Unless `\s` makes it appear as "cap"

Reply:
i forget why i did that but i know i had a reason hold on

(sorry for the javascript braces i thought i got them all)